packages/commerce-sdk-react/src/hooks/ShopperBaskets/config.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {ShopperBasketsTypes} from 'commerce-sdk-isomorphic'
+import {ApiClients, CacheUpdateMatrix} from '../types'
+import {CacheUpdate} from '../types'
+
+const updateBasketQuery = (basketId?: string): CacheUpdate => {
+    // TODO: we're missing headers, rawResponse -> not only {basketId}
+    const arg = {basketId}
+    return basketId
+        ? {
+              update: [['/baskets', basketId, arg]]
```

Comment:
I'm guessing it's still a WIP, but that query key format doesn't match the query key format used in the useQuery hooks.

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/ad4fac5da05d24197066cdf26ae7e9df8463bc24/packages/commerce-sdk-react/src/hooks/ShopperBaskets/query.ts#L31-L47

Reply:
Correct. In #959, I've settled on `[...path fragments, (path + query parameters)]`, and I've validated that all of the cache update logic follows the same format.